dashboard/app/views/layouts/application.html.haml

```diff
@@ -23,6 +23,8 @@
     -# data needed for dcdo.js. This tag is needed on every page because our translation code is
     -# making use of the dcdo feature.
     %script{data: {'dcdo': DCDO.frontend_config.to_json}}
+    -# auth key needed for amplitude analytics.
+    %script{data: {'amplitude-api-key': CDO.cdo_amplitude_api_key}}
```

Comment:
@rshipp I ended up changing the secrets key to `cdo_amplitude_api_key` so we don't conflict with Music Lab's